frame/governance-registry/src/lib.rs

```diff
@@ -1,8 +1,11 @@
 // TODO: move this pallet stuff into currency-factory (check if used on picasso and may need to
 // route from this pallet to factory)
-//! # Governance Registry Pallet
-//!
-//! Is used to add new assets into chain.
+// TODO: prove it supports `collective`(CallerOrigin) (which allows Council and other more
+// complicated custom)
```

Comment:
Can we move this TODO to a ticket?

Reply:
https://github.com/ComposableFi/composable/pull/1313